### PR TITLE
Add classification model MixNet-l

### PR DESCRIPTION
**Pull request by alexmasterdi**

### CODE REVIEW

File: models/public/mixnet-l/mixnet-l.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Channel order is `BGR`.
+
+## Output
+
+### Original Model
+
+Object classifier according to ImageNet classes, name - `logits`,  shape - `1,1000`, output data format is `B,C` where:
+
+- `B` - batch size
+- `C` - predicted probabilities for each class in [0, 1] range
```
Comment (eaidova) [inline models/public/mixnet-l/mixnet-l.md:62]: in [0, 1] range? If i right remember, we discussed that model return logits

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by eaidova** [comment]
@alexmasterdi please rebase your branch on open_model_zoo develop branch, your commit history looks nesty...

File: models/public/mixnet-l/mixnet-l.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Converted Model
+
+Object classifier according to ImageNet classes, name - `mixnet-l/mixnet_model/head/dense/BiasAdd/Add`,  shape - `1,1000`, output data format is `B,C` where:
```
Comment (eaidova) [inline models/public/mixnet-l/mixnet-l.md:66]: do you not try to provide --output=logits parameter to MO for preserving original model output name?

File: tools/accuracy_checker/configs/mixnet-l.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            type: INPUT
+
+    datasets:
+        
+      - name: imagenet_1000_classes
+        annotation_conversion:
+           converter: imagenet
+           annotation_file: val.txt
+        
+        data_source: ILSVRC2012_img_val
```
Comment (eaidova) [inline tools/accuracy_checker/configs/mixnet-l.yml:21]: please remove these sections, we apply dataset definitions for getting this info

File: tools/accuracy_checker/configs/mixnet-l.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        preprocessing:
+          - type: crop
+            central_fraction: 0.875
+            use_pillow: True
+          - type: resize
+            size: 224
+            use_pillow: True
+            interpolation: BICUBIC
+
+
```
Comment (eaidova) [inline tools/accuracy_checker/configs/mixnet-l.yml:84]: please remove trialing lines

File: models/public/mixnet-l/mixnet-l.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. The `mixnet-l` model is one of the
+[MixNet](https://arxiv.org/abs/1907.09595) models designed.
+This model was pretrained in TensorFlow\*.
+All the MixNet models have been pretrained on the ImageNet* image database.
```
Comment (eaidova) [inline models/public/mixnet-l/mixnet-l.md:8]:
```suggestion
All the MixNet models have been pretrained on the ImageNet\* image database.
```

**Comment by eaidova** [comment]
@alexmasterdi you also need:
1. Add model description to [index.md](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/public/index.md)
2. Add model to model.lst for [classification demo](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/demos/classification_demo/models.lst)
3. MoVe AC config to model directory and rename to accuracy-check.yml - see alexnet as example - https://github.com/openvinotoolkit/open_model_zoo/tree/develop/models/public/alexnet

File: models/public/mixnet-l/mixnet-l.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. The `mixnet-l` model is one of the
```
Comment (eaidova) [inline models/public/mixnet-l/mixnet-l.md:5]: what does mean "l" suffix in model name?

File: tools/accuracy_checker/configs/mixnet-l.yml [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    datasets:
+        
+      - name: imagenet_1000_classes
+        annotation_conversion:
+           converter: imagenet
+           annotation_file: val.txt
+        
+        data_source: ILSVRC2012_img_val
+        metrics:
```
Comment (eizamaliev) [inline tools/accuracy_checker/configs/mixnet-l.yml:22]: Swap `metrics` and `preprocessing` sections - that's way we normally do.

File: tools/accuracy_checker/configs/mixnet-l.yml [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+          - name: accuracy@top1
+            type: accuracy
+            top_k: 1
+          - name: accuracy@top5
+            type: accuracy
+            top_k: 5
+        
+        preprocessing:
```
Comment (eizamaliev) [inline tools/accuracy_checker/configs/mixnet-l.yml:75]: Order of resizing and cropping can affect accuracy. Do you sure that original model use such order?

File: models/public/mixnet-l/model.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+files:
+  - name: mixnet-l.tar.gz
+    size: 54302843
+    sha256: feb487ed76e46c4a19604172b4227a84db6bfd9449b022e310606525b2bb221f
+    source: https://storage.googleapis.com/cloud-tpu-checkpoints/mixnet/mixnet-l.tar.gz
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: mixnet-l.tar.gz
+model_optimizer_args:
```
Comment (eizamaliev) [inline models/public/mixnet-l/model.yml:32]: Does model have no mean or scale value?

File: models/public/mixnet-l/mixnet-l.md (hunk 6 above)
Comment (vladimir-dudnik) [reply models/public/mixnet-l/mixnet-l.md:5]: @eaidova according to description in original repository there are tree variations of mixnet model, MixNet-S, MixNet-M and MixNet-L. So, I'd think the postfix `l` (it is lower case letter L, not a number one) is stands for MixNet-L model.
